package.json

```diff
@@ -26,18 +26,18 @@
   },
   "homepage": "https://github.com/gj/fastify-ws#readme",
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "fastify": "^1.7.0",
+    "eslint": "^5.14.1",
+    "fastify": "^2.0.0",
```

Comment:
I wonder if we should change this to something like `"fastify": "1.7.0 - 2"` to bring these package bumps to users of Fastify 1.x as well. If there's no breaking change with the Fastify API that we're interfacing with (need to test this out later), I think I'd prefer to maintain compatibility.